src/packages/client/src/generation/TSClient/TSClient.ts

```diff
@@ -262,7 +262,27 @@ ${prismaEnums.join('\n\n')}
  */
 
 ${this.dmmf.inputObjectTypes.prisma
-  .map((inputType) => new InputType(inputType, collector).toTS())
+  .reduce((acc, inputType) => {
```

Comment:
Looks like a lot, maybe could use a comment so it's easier to read/understand the code?